.ci/checkTranslator.sh

```diff
@@ -40,19 +45,28 @@ notOneTranslatorID () {
 nonUniqueTranslatorID () {
     local dir id duplicateIds
     dir=$(dirname "$TRANSLATOR")
-    id=$(grep -r '"translatorID"' "$TRANSLATOR" \
-        | sed -e 's/[" ,]//g' -e 's/^.*://g')
+    id=$(grepTranslatorId "$TRANSLATOR")
     if [[ -n "$id" ]];then
-      duplicateIds=$(grep -r '"translatorID"' "$dir"/*.js \
-              | sed -e 's/[" ,]//g' -e 's/^.*://g' \
-              | sort | uniq -d)
+      duplicateIds=$(grepTranslatorId "$dir"/*.js | sort | uniq -d)
```

Comment:
Actually, this will be performed for each translator, which seems to be then too much, because we already know the specific `id` to look for. I would suggest something like
```bash
duplicateIds=$(grep "$id" | uniq -d)
```
which can then be tested to being empty.